idea/src/org/jetbrains/kotlin/idea/parameterInfo/KotlinFunctionParameterInfoHandler.kt

```diff
@@ -254,6 +254,10 @@ abstract class KotlinParameterInfoWithCallHandlerBase<TArgumentList : KtElement,
         return buildString {
             if (named) append("[")
 
+            parameter.annotations.getAllAnnotations().forEach {
+                it.annotation.fqName?.shortName()?.asString()?.let { append("@$it ") }
```

Comment:
Please, extract fqName to variable to reduce safe-call count

Reply:
How about this?

```kotlin
it.annotation.fqName?.let { append("@${it.shortName().asString()} ") }
```

Reply:
Cool!